nrrd-design-system/components/colors/supporting-palette/README.md

```diff
@@ -0,0 +1,28 @@
+*"Accessible” colors can be used in combination with white text or backgrounds, since the contrast ratio between the color and white is at least 4.5:1, the [WCAG 2.0 level AA](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast) compliance ratio for color contrast.*
+
+The supporting color palette is composed of lightest and darkest variations of the primary color palette. A broad supporting color palette adds nuance and depth to illustrations, data visualizations, and complex components.
+
+## How colors are used
+
+### Blacks and grays
+Cool grays are used widely in the site, predominantly for large background fills.
+- `$black`: `h1`, `h3`, and `h4` header text
+- `$gray-darkest`: `h5` header text
+- `$gray-dark`: the text on disabled buttons
+- `$gray`: maps, as rules with custom headline styles on [Revenue pages](https://revenuedata.doi.gov/how-it-works/offshore-oil-gas/)
```

Comment:
I recommend we change the wording for this link. "Revenue pages" could apply to many different pages (or portions of pages) on the site. Internally, we call them "diving graphics," but that's esoteric for those outside the org. 

What do you think about "revenue-process pages"? Or "from-lease-to-revenue pages"?

Reply:
Makes sense to me, if one isn't necessarily more used by the other at this point, I'd lean toward "revenue-process pages" because it sounds more natural to start with a noun in the title than a preposition. And it IS important to use what the internal team would recognize, you're the primary audience and users of this tool! What do you think of `on revenue-process pages with diving graphics` ?